src/ScrollList/index.tsx

```diff
@@ -35,14 +35,15 @@ export const CustomScrollContainer = React.forwardRef<HTMLDivElement, IFixedSize
 interface IFixedSizeListProps extends Omit<ReactWindow.FixedSizeListProps, 'height' | 'width'> {
   className?: string;
   maxRows?: number;
+  addOffset?: boolean;
```

Comment:
I suggest changing this to `offset` so we can choose to add or remove some height from the list. It would default to `0`. Thoughts?

```suggestion
  offset?: number;
```